packages/esm-tb-app/src/views/dashboard/patient-list-tabs/tb-patient-list-tabs.component.tsx

```diff
@@ -49,8 +50,14 @@ function TbHomePatientTabs() {
           {
             key: 'nextAppointmentDate',
             header: t('appointmentDate', 'Appointment Date'),
-            getValue: ({ latestEncounter }) => {
-              return getObsFromEncounter(latestEncounter, config.obsConcepts.nextAppointmentDate, true);
+            getValue:  (patient) => {
+              const patientLatestExtraEncounters = patient.latestExtraEncounters
+              if (patientLatestExtraEncounters && patientLatestExtraEncounters.length) {
+                const latestFollowUpEncounter = patientLatestExtraEncounters[0];
```

Comment:
@kajambiya are you sure the encounter you need will always be at position `zero`? What happen if someone changes the order? This code will break

Reply:
I had done it like that because in the list of extraAssociatedEncounterTypes we had only passed by one item but I'm going to implement @pirupius's solution. If I find by the encounterType, even if the extraAssociatedEncounterTypes array has more than one item and the order changes, the code won't break.